common/app/views/fragments/omnitureScript.scala.html

```diff
@@ -0,0 +1,28 @@
+@(item: Option[model.MetaData])
+
+@import conf.Switches.InlineOmnitureScript
+@import conf.Static
+@import conf.Configuration
+@import views.support.OmnitureAnalyticsAccount
+
+@item.map{ item =>
+    <script>
+        @* must always be set before the Omniture file is parsed *@
+        window.s_account = '@OmnitureAnalyticsAccount(item)';
+    </script>
+}
+
+@if(InlineOmnitureScript.isSwitchedOn) {
+    <script>
+        try {
+            @Html(Static.js.omnitureJs)
+        } catch (e) {
+            (new Image()).src = '@{Configuration.debug.beaconUrl}/count/omniture-library-error.gif';
+        }
+    </script>
+} else {
+    @* hardly ever changes, so extremely cacheable *@
+    @* unfortunately we cannot put 'defer' on it because we must be able to guarantee this script has run before other
+    scripts are called, hence the ability to switch between this and an inline version for testing *@
+    <script id="omniture" src="@Static("javascripts/vendor/omniture.js")"></script>
```

Comment:
I fear this will have a drastic effect on our speed index/first paint time, even if only for a short while.

Maybe you could do something like this in the Omniture dependent code:

``` js
if (switch) {
    require('omniture', function (s) {
        doThing(s);
    })
} else {
    doThing(s);
}
```